appium-demos/pom.xml

```diff
@@ -51,6 +51,24 @@
       <version>5.5.0-M1</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.testng</groupId>
```

Comment:
We should stick with either JUnit or Testng. Having both on the classpath leads to all sorts of issues.